src/GRA.Controllers/ProfileController.cs

```diff
@@ -1234,6 +1247,94 @@ public async Task<IActionResult> History(int page = 1)
             return View(viewModel);
         }
 
+        public async Task<IActionResult> Badges(int page = 1)
+        {
+            User user = await _userService.GetDetails(GetActiveUserId());
```

Comment:
Do we need to look up the user here? It seems like maybe we already have the needed user's information stored in the session for this method.

Reply:
This was just the general way I saw in the GRA to retrieve the current user.
We need to have at least the user's Id and the user's HouseholdHeadUserId